test_predicates.py

```diff
@@ -39,35 +41,35 @@ def test_equal(self):
                                matrix([0, 1, -1], # paranoia seems wise
                                       [-2, 0, 0],
                                       [0, 0, 0])))
-        self.assertFalse(equal(vector(0, 1, 2), vector(1, 2)))
+        self.assertFalse(equal(Vector(0, 1, 2), Vector(1, 2)))
 
 
-    def test_as_columns(self):
-        """Test the thing that joins column vectors into a matrix."""
-        target = matrix([1, 2, 3],
-                        [4, 5, 6],
-                        [7, 8, 9])
-        col1 = vector(1, 4, 7)
-        col2 = vector(2, 5, 8)
-        col3 = vector(3, 6, 9)
-        self.assertTrue(equal(target, as_columns(col1, col2, col3)))
+    # def test_as_columns(self):
+    #     """Test the thing that joins column vectors into a matrix."""
+    #     target = matrix([1, 2, 3],
+    #                     [4, 5, 6],
+    #                     [7, 8, 9])
+    #     col1 = Vector(1, 4, 7)
+    #     col2 = Vector(2, 5, 8)
+    #     col3 = Vector(3, 6, 9)
+    #     self.assertTrue(equal(target, as_columns(col1, col2, col3)))
```

Comment:
I'd say either delete this, or at least give some justification as to why it is all commented out.